monailabel/endpoints/datastore_review.py

```
@@ -0,0 +1,287 @@
+        )
+
+    return {"status": "success", "image_id": image, "versions": versions}
+
+
+@router.get("/report", summary=f"{RBAC_USER}Generate review report")
+async def api_review_report(
+    fmt: str = "json",
+    reviewer: Optional[str] = None,
+    date_range: Optional[str] = None,
+    tag: str = DefaultLabelTag.FINAL.value,
+    user: User = Depends(RBAC(settings.MONAI_LABEL_AUTH_ROLE_USER)),
```

Ruff (0.15.21)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)